src/components/Dashboard/PositionsTable/PositionsTable.js

```diff
@@ -214,10 +227,13 @@ const PositionsTable = (props) => {
 
     if (type === "closed") {
       dataTable = composeClosePositionsDataTable();
+      setCloseCount(filterPastDayPositions(positionsAll));
     } else if (type === "log") {
       dataTable = composeLogPositionsDataTable();
+      setLogCount(positionsAll.length);
     } else if (type === "open") {
       dataTable = composeOpenPositionsDataTable();
+      setOpenCount(positionsAll.length);
```

Comment:
Looks like it can be removed since there is a 2nd setLogCount below.

Reply:
@cwagner22  the one below is for profile positions and this one is for dashboard positions.

Reply:
I don't understand, you are calling `setOpenCount(positionsAll.length);` twice right?

Reply:
My bad, the brace closing the block was collapsed so I didn't see that hehe.